roles/network_plugin/cilium/templates/cilium/config.yml.j2

```
@@ -144,6 +144,11 @@ data:
   # Bandwidth Manager requires a v5.1.x or more recent Linux kernel.
 {% if cilium_enable_bandwidth_manager %}
   enable-bandwidth-manager: "true"
+  # Enable BBR for the bandwidth manager
+  # BBR for Pods requires a v5.18.x or more recent Linux kernel.
+{% if cilium_enable_bbr %}
+  enable-bbr: "true"
+{% endif %}
```

```suggestion
  enable-bbr: "{{ cilium_enable_bbr | lower }}"
```
Unless there is a downside to explicitly set if to false ?

---

https://github.com/kubernetes-sigs/kubespray/pull/11730#issuecomment-2505684630